pkg/cluster/controllers/storage/ingress.go

```diff
@@ -105,31 +105,52 @@ func (m *MinIOController) cleanupIngress(ctx context.Context, harborcluster *goh
 	return minioUnknownStatus(), nil
 }
 
-func (m *MinIOController) generateIngress(ctx context.Context, harborcluster *goharborv1.HarborCluster) *netv1.Ingress { // nolint:funlen
-	var tls []netv1.IngressTLS
-
+func (m *MinIOController) getMinioIngressAnnotations(harborcluster *goharborv1.HarborCluster) map[string]string {
+	istls := false
 	if harborcluster.Spec.Storage.Spec.MinIO.Redirect.Expose != nil &&
```

Comment:
```suggestion
	if harborcluster.Spec.Storage.Spec.MinIO.Redirect.Expose != nil
```
```
isEnableExpose := false
if harborcluster.Spec.Storage.Spec.MinIO.Redirect.Expose != nil { sEnableExpose = ture }
```
how about using any variable like 'isEnableExpose' to mark whether `Expose` or not, instead of checking it every time.